cmd/virt-launcher/virt-launcher.go

```diff
@@ -421,6 +421,9 @@ func main() {
 		panic(err)
 	}
 
+	// This is needed for swtpm_setup, when running as the qemu user (swtpm_user = "qemu" in qemu.conf)
+	os.Chown("/var/lib/swtpm-localca", 107, 107)
```

Comment:
Please use an OwnershipManager as non-root can't do this.

Reply:
Good point, but might just switch to user config, as per @vasiliy-ul suggestion, so we won't need this.